back-end/legeno-around-here/src/main/java/wooteco/team/ittabi/legenoaroundhere/service/MailAuthService.java

```diff
@@ -0,0 +1,89 @@
+package wooteco.team.ittabi.legenoaroundhere.service;
+
+import java.util.Random;
+import javax.mail.MessagingException;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import wooteco.team.ittabi.legenoaroundhere.domain.user.Email;
+import wooteco.team.ittabi.legenoaroundhere.domain.user.User;
+import wooteco.team.ittabi.legenoaroundhere.domain.user.mailauth.MailAuth;
+import wooteco.team.ittabi.legenoaroundhere.dto.MailAuthCheckRequest;
+import wooteco.team.ittabi.legenoaroundhere.dto.MailAuthCreateRequest;
+import wooteco.team.ittabi.legenoaroundhere.exception.FailedSendMailException;
+import wooteco.team.ittabi.legenoaroundhere.exception.NotExistsException;
+import wooteco.team.ittabi.legenoaroundhere.exception.WrongUserInputException;
+import wooteco.team.ittabi.legenoaroundhere.repository.MailAuthRepository;
+import wooteco.team.ittabi.legenoaroundhere.repository.UserRepository;
+import wooteco.team.ittabi.legenoaroundhere.utils.MailHandler;
+
+@Slf4j
+@Transactional
+@Service
+@AllArgsConstructor
+public class MailAuthService {
+
+    private static final int AUTH_NUMBER_MIN = 100000;
```

Comment:
`_`를 써주면 가독성을 확보할 수 있죠!

```suggestion
    private static final int AUTH_NUMBER_MIN = 100_000;
```

Reply:
👍